### PR TITLE
Fix documentation typos

### PR DESCRIPTION
**Pull request by cjrace**
# Brief overview of changes

Adding the spelling package into the checks on the package and fixed up a few typos I spotted.

## Why are these changes being made?

Noticed a couple of typos when promoting the external_link() function and figured I'd quickly fix them, as well as add the spelling checks to prevent more going forwards.

## Detailed description of changes

No extra detail needed

## Additional information for reviewers

Made it a dev version as it's not worth a version bump (only the typos are user facing, the rest is all behind the scenes) and these can just go into our next update.

## Issue ticket number/s and link

No related issue number

### CODE REVIEW

File: tests/spelling.R [hunk 1]
```diff
@@ -0,0 +1,3 @@
+if(requireNamespace('spelling', quietly = TRUE))
```
[the remark this answers is not in the document]
Comment (cjrace) [reply tests/spelling.R]: Fixed

[the remark this answers is not in the document]
Comment (cjrace) [reply tests/spelling.R]: Fixed

File: R/cookies.R [hunk 2]
```diff
@@ -342,7 +342,7 @@ Shiny.addCustomMessageHandler('analytics-consent', function(msg){
 #' "ABCDE12345"
 #'
 #' @family cookies
-#' @return a html div, containing standard cookises content for a public R
+#' @return a html div, containing standard cookies content for a public R
```
Comment (jen-machin) [inline R/cookies.R:345]: I think HTML should be in caps - it's not in the style guide but this is consistent with how it's used elsewhere

**Review by jen-machin — COMMENTED** [review]
one tiny change!

File: R/cookies.R (hunk 2 above)
Comment (cjrace) [reply R/cookies.R]: Fixed this, and a few other instances in my last commit

**Pull request merged**